### PR TITLE
Specs for fetching thumbnails

### LINKED ISSUE #4: Add API docs for fetching thumbnails
Thumbnail caching functionality and API were added to the FS in this pr https://github.com/amahi/amahi-anywhere-fs/pull/20, its corresponding docs need to be updated.

### PR DESCRIPTION
**Pull request by vishwasmittal**
Added specs for fetching thumbnails. Fixes #4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -209,19 +209,24 @@ To authenticate, the client app has to issue a `POST` to `/auth` with an `applic
   * The `mime_type` element represents the expected [MIME type](https://en.wikipedia.org/wiki/Internet_media_type) of the file based on the extension (the actual file could have some other data inside it)
   * The `mtime` element is the modification time of the entry
   * The `size` element contains the size of the file at the time of retrieval of the directory in bytes. It is a very large unsigned integer (uint64)
-  * If the entry the case of a directory entry, the `mime_type` is set to `text/directory`, and the `size` is 0
+  * The `cache` element contains the details about the thumbnail cache for any media item (photos and/or videos). It is a json object with the following fields:
```
Comment (csoni111) [inline README.md]: Do we make thumbnails for video files too?

Comment (vishwasmittal) [reply README.md]: Currently, the library we are using is unable to do so but we certainly can other implementations for the same.

Comment (csoni111) [reply README.md]: then replace this `any media item (photos and/or videos)` by `any image file`

**Review by csoni111 — APPROVED** [review]
@cpg please review and merge

File: README.md [hunk 2]
```diff
@@ -209,19 +209,24 @@ To authenticate, the client app has to issue a `POST` to `/auth` with an `applic
...
-          { "name": "image.png", "mime_type": "image/png", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 28517360 },
-          { "name": "movie.mkv", "mime_type": "video/webm", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 45678 },
-          { "name": "movie.mp4", "mime_type": "video/mp4", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 12345 }
-          { "name": "large.iso", "mime_type": "application/octet-stream", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 32839273198 }
+          { "name": "dir", "mime_type": "text/directory", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 0, "cache": {"status": false} },
+          { "name": "README.txt", "mime_type": "text/plain", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 12345, "cache": {"status": false} },
+          { "name": "music.mp3", "mime_type": "audio/mpeg", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 23456, "cache": {"status": false} },
+          { "name": "image.jpg", "mime_type": "image/jpeg", "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 34567, , "cache": {"status": true, "mtime": "Sat, 17 Aug 2013 02:38:32 GMT", "size": 4567, "endpoint": "/cache?s=<sharename>&p=<file-path>"} },
```
Comment (csoni111) [inline README.md:225]: I missed this, there are two commas here `"size": 34567, , "cache":`. Please make a new pr and rectify this. @vishwasmittal

**Pull request merged**